client/packages/programs/src/JsonForms/common/components/Select.tsx

```diff
@@ -131,28 +207,49 @@ const UIComponent = (props: ControlProps) => {
           value={value}
           onChange={onChange}
           filterOptions={filterOptions}
+          getOptionLabel={getOptionLabel}
           renderOption={(props, option, { inputValue }) => {
-            const matches = match(option.label, inputValue, {
-              insideWords: true,
-            });
-            const parts = parse(option.label, matches);
-
-            return (
-              <li {...props}>
-                <div>
-                  {parts.map((part, index) => (
-                    <span
-                      key={index}
-                      style={{
-                        fontWeight: part.highlight ? 600 : 400,
-                      }}
-                    >
-                      {part.text}
-                    </span>
-                  ))}
-                </div>
-              </li>
-            );
+            const parts = getHighlightParts(value, option.label, inputValue);
+
+            if (schemaOptions?.multiColumn) {
+              const descriptionParts = getHighlightParts(
+                value,
+                option.description,
+                inputValue
+              );
+              const rightParts = getHighlightParts(
+                value,
+                option.right,
+                inputValue
+              );
+              return (
+                <li {...props} key={option.value}>
+                  <span
+                    style={{ whiteSpace: 'nowrap', width: 100, minWidth: 50 }}
+                  >
+                    <TextHighlight {...props} parts={parts} />
+                  </span>
+                  <span style={{ whiteSpace: 'nowrap', width: 500 }}>
```

Comment:
Also, do you think the widths should be relative to the column widths specified in `styleConstants.ts`?

So instead of `500` here, it would be `FORM_LABEL_COLUMN_WIDTH - 50` ? (with an additional adjustment for the right parts)

Reply:
thanks I set whiteSpace to 'normal' ('wrap' was an error for me...) 

FORM_LABEL_COLUMN_WIDTH is percent string here so its a bit tricky. Since its ok right now I left it as is.